docs/starters.yml

```diff
@@ -755,3 +755,19 @@
     - Responsive Design, optimized for Mobile devices
     - styled-components
     - JavaScript Standard Style
+- url: https://happy-pare-dff451.netlify.com/
+  repo: https://github.com/fhavrlent/gastby-contentful-typescript-starter
+  description: Your starter description
+  # These correspond to the category filters in the library
+  # Make an effort to use the existing tags, and add more if needed!
+  tags:
+    - Contentful
+    - TypeScript
+    - Emotion
```

Comment:
Please use the tag `Styling:CSS-in-JS` instead of `Emotion`, fill out the `description` and remove the `#` comments. Other than that it's good to go imo!